fact_extractor/plugins/unpacking/intel_hex/install.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-
-sudo -EH pip3 install --upgrade intelhex
```

Comment:
Is there a reason this setup was moved to the `install/unpacker.py`, while others weren't?

Reply:
Question also applies to: mikrotik, kernel, parts of dji, raw, and so on..

Reply:
I only moved python package installation to `install/unpacker.py` to be able to adjust the installation in case of a venv without having to do this in every single `install.sh` (and I also didn't want to convert every installation to a `install.py` but maybe this is a thing that we should do at some point :sweat_smile: )

Reply:
I do understand that, it's just one more thing that binds plugins to the _core_. I'd say if we go down this road we can drop the plugin `install.sh` files altogether, but I guess for the moment we can keep it like this and ignore the ambiguity...

Reply:
I would prefer a solution closer to what we did in FACT_core (setup.py/requirements.txt/apt_packages.txt/etc.) but this is absolutely out of scope for this PR. I don't know how relevant custom plugins are for the extractor, though.